hocr-check

```diff
@@ -57,9 +57,17 @@ def mostly_nonoverlapping(boxes,significant_overlap=0.2):
 ### main
 ################################################################
 
-if len(sys.argv)<1:
+def print_usage():
     print "usage:",sys.argv[0],"[-o] file.html"
+
```

Comment:
Usage: ...

It's common practice to start help text with upper case, so I suggest to change it here to upper case, too.